### ISSUE TITLE
Auto-start with test page in dev

### ISSUE THREAD

**Issue by tom2strobl** [extension-js/extension.js#216]
Hi there,

I would like to automatically open up a test page in chrome **only** in dev when running my _dev_ command (so no manifest stuff). I'll serve the page myself locally (localhost:3008), but I'm struggling to find a way to start the browser with the page opened.

I tried all of these (separately) in `extension.config.js` but none of them seemed to work:

```typescript
// https://www.chromium.org/administrators/configuring-other-preferences/
browser: {
  chrome: {
    browserFlags: ['--startup-page="https://localhost:3008'],
    preferences: {
      homepage: 'https://localhost:3008',
      homepage_is_newtabpage: true,
      session: {
        startup_urls: ['https://localhost:3008']
      }
    }
  }
}
```

**Comment by cezaraugusto** [comment]
hey @tom2strobl, great to see you here again!

yeah, `extension.config.js` should provide a "startingUrl" option for each browser field and we should also have a  `--start-url` flag or something so users can `extension dev --starting-url=https://example.com`. Documentation is in its early days so these are likely not documented.

I'll be checking on this at some point for sure but if you're feeling adventurous I can definitely help reviewing a PR to either fix what is supposed to work or just add it to the docs. 

btw added you as a collaborator to the project in case you want to get more involved. thanks for the recent reports!

**Comment by tom2strobl** [comment]
Yeah it's really extremely low priority, so don't feel obligated to take a look anytime soon. Thanks for the invite, I'll see if I can contribute anything worthwile

**Comment by cezaraugusto** [comment]
hey @tom2strob, happy to share this is available in v2.0.0-beta.3, both command-line (`extension@latest dev --starting-url="XXX"`) and via `extension.config.js`. let me know you have a chance to try it out!

**Comment by tom2strobl** [comment]
Busy times but can now confirm works like a charm 🚀
